### ISSUE TITLE
produce sample speaker audio for default install

### ISSUE THREAD

**Issue by hburgund** [roundware/roundware-server#268]
To facilitate testing, it would be ideal for the default RW install to have several speakers automatically created locally from audio files included in the distro.

The best situation for testing would be several (say three) simple pleasant audio files of single constant notes that are easily distinguishable from each other even when playing simultaneously. Perhaps a 1-3-5 or maybe the same note in three registers. These files will be turned into streams automatically (see #131) and then using the Listening Map, testing would be vastly easier and more accurate.

**Comment by 13rac1** [comment]
Maybe optional? Like this https://github.com/roundware/roundware-server/blob/develop/scripts/add-test-audio.py

**Comment by hburgund** [comment]
Yeah, good idea. Probably the best situation would be to have one speaker automatically created locally (just as we have now, but with local stream, not external stream) and then have the option through a script to add a few more test speakers.

Once #131 is fixed, this should be fairly straight-forward, I would think.
